aiken-contracts/fractional/scripts/batcher_v3/src/queue_purchase.py

```diff
@@ -26,35 +27,37 @@ def build_tx(sale_info, queue_info, batcher_info, constants: dict) -> Tuple[dict
     batcher_pkh = constants['batcher_pkh']
 
     # HARDCODE FEE FOR NOW, NEED WAY TO ESITMATE THESE UNITS BETTER
-    FEE = 600000
+    FEE = 505549
```

Comment:
We should be able to calculate these directly when we build the rust batcher.

Reply:
Nice! Yeah, hard coding for now as it is a limitation of my [tx simulation](https://github.com/logicalmechanism/tx_simulation) when tx chaining. I did do proper tests and the fee for the swap is more like 400,000 Lovelace and the refund is around 300,000 Lovelace so being able to calculate the real values will be worth it.